classes/converter.php

```diff
@@ -126,10 +131,19 @@ public static function are_requirements_met() {
      * @return  this
      */
     public function start_document_conversion(\core_files\conversion $conversion) {
-        global $CFG;
+        global $CFG, $SITE;
+        $logger_installed = $this->has_pxsdk_logger_installed();
+
+        if ($logger_installed) {
+            $factory = \local_pxsdk\app\v10\factory::make();
+
+            $index = $factory->logger()->index()->default_elasticsearch_logger_index(false);
+            $logger_repo = $factory->logger()->repository('elasticsearch', $index);
+        }
```

Comment:
If all log parts are extracted into a method then this should no longer be relevant.